README.md

```diff
@@ -238,7 +204,7 @@ package rules.my_simple_rule
 resource_type = "aws_ebs_volume"
 
 # Controls.
-controls = {"CIS_1-16"}
+controls = {"CIS-AWS_v1.2.0_1.16"}
```

Comment:
Is this still the proper syntax for specifying a control, since the metadata looks a little different now?